### ISSUE TITLE
Collisions apply a linear impulse to the center of mass

### ISSUE THREAD

**Issue by randomguyeatspotato** [synthein/synthein#188]
This should be applied to the parts location.

**Comment by xordspar0** [comment]
Are you sure? How can I verify this?

**Comment by xordspar0** [comment]
We talked about this and I realized that I misunderstood. I thought we were just letting the physics engine handle the collision based on the physical properties of the objects. However, when there's a collision we manually apply a linear impulse on the objects colliding.

```lua
	function self.userData:collision(fixture, otherFixture, sqVelocity, pointVelocity)
		[...]
		object:damage(otherFixture, damage)
		local body = fixture:getBody()
		local mult = -damage

		if mult < -10 then mult = -10 end
		mult = mult / 10
		[...]
		body:applyLinearImpulse(xI * mult, yI * mult)
	end
```

Maybe instead of applying an impulse like this we could just make the objects more bouncy. Box2d already knows how to distribute forces in a collision. I'm not sure we should add our own forces to the mix.
